gruntfile.js

```diff
@@ -1,45 +0,0 @@
-module.exports = function(grunt) {
```

Comment:
Well, it's two files that we need to bump. And I thought Grunt with Webpack is too much. They share a lot of the functionality.